release-calendar/src/client/components/Tooltip.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {EventApi, View} from '@fullcalendar/core';
+import {Hook} from './Hook';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export const Tooltip = (arg: {
```

Comment:
The way this component is created looks different than other components - i.e. why is this a const instead of a class of type `React.Component`?

Reply:
Because it's not really a component in itself. It's replacing the existing component events from the calendar with the component I build in the hook. https://reactjs.org/docs/react-dom.html

Reply:
I see. The docs say "Most of your components should not need to use this module." hmmm... we can punt this for now.